PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -1,90 +1,149 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.IO;
 
-namespace PrincessBrideTrivia.Tests
-{
-    [TestClass]
-    public class ProgramTests
-    {
-        [TestMethod]
-        public void LoadQuestions_RetrievesQuestionsFromFile()
-        {
-            string filePath = Path.GetRandomFileName();
-            try
-            {
-                // Arrange
-                GenerateQuestionsFile(filePath, 2);
 
-                // Act
-                Question[] questions = Program.LoadQuestions(filePath);
 
-                // Assert 
-                Assert.AreEqual(2, questions.Length);
-            }
-            finally
-            {
-                File.Delete(filePath);
-            }
-        }
+namespace PrincessBrideTrivia.Tests;
```

Comment:
```suggestion
namespace PrincessBrideTrivia.Tests {
 ...
```

It looks like this caused your diff issues. Is there a reason you changed this up?